File: LeetCode/2114. Maximum Number of Words Found in Sentences/solution.py
"""
Runtime: 36 ms
Memory Usage: 15.1 MB
"""
class Solution:
    def mostWordsFound(self, sentences: List[str]) -> int:
        ans = 0
        for sentence in sentences:
            w = sentence.count(' ') + 1
            if not ans or w > ans:
                ans = w 
        return ans

File: LeetCode/2114. Maximum Number of Words Found in Sentences/solution.cpp
/**
 * Runtime: 12 ms
 * Memory Usage: 9.3 MB 
 */
class Solution {
public:
    int mostWordsFound(vector<string>& sentences) {
        int ans = 0;
        for (auto &sentence: sentences) {
            int s = 1;
            for (auto &c: sentence) {
                if (c == ' ') ++ s;
            }
            if (ans == 0 || s > ans) {
                ans = s;
            }
        }
        return ans;
    }
};
